### DISCUSSION TITLE
How do I get the DSO quality images that are on the web version in the desktop version?

### DISCUSSION THREAD

**Discussion by rrbailey89** [Stellarium/stellarium#2733] [Q&A] [answered]
Title says it all, I notice that the web version of stellarium has higher quality images for DSO and the desktop version doesn't seem to have these by default. Is there a setting that will enable this?

**Comment by gzotti** [comment, marked as answer]
You can enable the same background (DSS) in the desktop version.

**Reply by rrbailey89** [reply]
How do I do that?

**Reply by gzotti** [reply]
By enabling display of the DSS button (if necessary) and then clicking on it. The User Guide has more on this.
